### DISCUSSION TITLE
No traceId when I use ServerHttpResponseDecorator

### DISCUSSION THREAD

**Discussion by zhangzx1996** [apache/skywalking#12574] [Q&A] [answered]
I want to change response ,  I used WebFluxSkyWalkingOperators.continueTracing . but the traceId is dispear when execute  ServerHttpResponseDecorator.
![image](https://github.com/user-attachments/assets/51a67bf4-9268-4418-a58a-9f5e4b73ec6a)
![image](https://github.com/user-attachments/assets/15add3ca-f869-4b1d-8e8d-2b1c5906f11c)

**Comment by wu-sheng** [comment, marked as answer]
That means this is another thread. About why, you need to get into webflux codebase by yourself.
We don't touch that part.

**Reply by zhangzx1996** [reply]
OK, Very thanks!
